addons/mail_bot/static/src/owl/components/notification_list/notification_list.scss

```diff
@@ -0,0 +1,11 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_NotificationList_NotificationRequest {
```

Comment:
`_notificationRequest` (start with smallCase for naming parts of a Component)

Reply:
ok